2-Draft-Accepted/RFCNNN-HelpSystem-As-A-Module.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: RFC<four digit unique incrementing number assigned by Committee, this shall be left blank by the author>
+Author: Aditya Patwardhan
+Status: Draft
+SupercededBy: <link to another RFC>
+Version: 0.1
+Area: Help System
+Comments Due: 08/15/2017
+Plan to implement: Yes
+---
+
+# Help System as a standalone PowerShell Module  
+
+This RFC proposes to remove the Help System from System.Management.Automation.dll and make it a standalone PowerShell module.
+
+## Motivation
+
+Currently, help system is part of System.Management.Automation.dll. 
+Help system is used predominantly in interactive sessions and hence not required for automation scenarios.
+Since, it is part of System.Management.Automation.dll it is always loaded with PowerShell.
+Help system is sizeable piece of code, which increases the runtime footprint of the process.
+This RFC proposes to make the help system a PowerShell module, so it is loaded only when used.
```

Comment:
If you have thoughts on public api's, I think it's worth including here or note that it'll be in a separate RFC